### PR TITLE
podman cli and ansible ssh long fqdn fix 

### PR DESCRIPTION
**Pull request by XDsubhash**
* Added podman support to umb triggered suites (ansible, ansible container)
  * rgw suite needs couple fixes in ceph-qe-scripts to pass ,will do it in another pr
  * rados suite, https://bugzilla.redhat.com/show_bug.cgi?id=1811910 https://bugzilla.redhat.com/show_bug.cgi?id=1632494 , no authentic way 
* Added rhel 7.8 image yaml
* Fix unix domain too long ssh error by setting ansible env variable

### CODE REVIEW

**Comment by XDsubhash** [comment]
#67 is little behind. https://ceph-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/RHCS%20Sanity/job/rhceph-sandbox-1/12/console

#86 commented the TCs untill fixed

File: ceph/ceph.py [hunk 1]
```diff
@@ -414,11 +414,15 @@ def setup_insecure_registry(self):
...
         """
         if self.containerized and self.ansible_config.get('ceph_docker_registry'):
-            insecure_registry = '{{"insecure-registries" : ["{registry}"]}}'.format(
-                registry=self.ansible_config.get('ceph_docker_registry'))
-            logger.warn('Adding insecure registry:\n{registry}'.format(registry=insecure_registry))
-            for node in self.get_nodes():
-                node.write_docker_daemon_json(insecure_registry)
+            logger.warn('Adding insecure registry:\n{registry}'
+                        .format(registry=self.ansible_config.get('ceph_docker_registry')))
+            # for node in self.get_nodes():
```
Comment (vasukulkarni) [inline ceph/ceph.py:419]: In the follow on PR, please remove the unneeded comments.

File: suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml [hunk 2]
```diff
@@ -55,38 +55,38 @@ tests:
...
-   - test:
-      name: rbd cli import_export
-      module: rbd_system.py
-      config:
-            test_name: cli/rbd_cli_import_export_diff.py
-            branch: master
-      polarion-id: CEPH-83572723
-      desc: CLI validation for import, export and diff related commands
+  #  - test:
+  #     name: rbd cli image
```
Comment (vasukulkarni) [inline suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml:59]: same, lets remove the unneeded comments

Comment (vasukulkarni) [reply suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml:59]: Lets take this in next PR.

File: ceph/ceph.py (hunk 1 above)
Comment (XDsubhash) [reply ceph/ceph.py:419]: sure, for now the registry we use dont need to added.

File: suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml (hunk 2 above)
Comment (XDsubhash) [reply suites/nautilus/ansible/sanity_containerized_ceph_ansible.yaml:59]: okay , once #86 is fixed will enable them

**Pull request merged**